float64 x_act
float64 y_act
float64 z_act
float64 roll_act
float64 pitch_act
float64 yaw_act
float64 u_act
float64 v_act
float64 w_act
float64 p_act
float64 q_act
float64 r_act
float64 adap
float64 delta_v_act
float64 delta_m_act
float64 delta_s_act
float64 zend
float64 thetaend
float64 tf
float64 tend
float64 t0
float64 y_aug

